python/cudf_polars/cudf_polars/experimental/base.py

```diff
@@ -207,14 +213,19 @@ def unique_stats(self, *, force: bool = False) -> UniqueStats:
         force
             If True, return unique-value statistics even if the column
             wasn't marked as needing unique-value information.
+
+        Notes
+        -----
+        This method uses the statistics collection stream.
         """
+        stream = get_stream_for_stats()
```

Comment:
Something is wrong here: this function accepts a `stream` so why is it being added here?

Regardless, the parquet stats collector needs a `stream` when sampling row groups, and so it needs to be added to the base class and threaded through all the way to the method that does the `read_parquet`.